docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+db:
+  image: postgres
+  ports:
+    - "5432"
+web:
+  build: .
+  command: >
+    /bin/bash -c '
+    sleep 3 &&
+    python manage.py migrate --noinput &&
+    python manage.py runserver 0.0.0.0:8077'
+  volumes:
+    - .:/src
+  environment:
+    PORT: 8077
+    DATABASE_URL: postgres://postgres@db:5432/postgres
+  ports:
+    - "8077:8077"
+  links:
+    - db
+dredd:
+  image: ccxcon_web
+  command: /bin/bash -c 'sleep 4 && dredd'
```

Comment:
Upgrading to docker-compose 1.5.0 should fix this. If not, let me know.

On Thu, Nov 12, 2015 at 3:41 PM, George Schneeloch <notifications@github.com
> wrote:

> In docker-compose.yml
> <https://github.com/mitodl/ccxcon/pull/7#discussion_r44711448>:
>
> > +    /bin/bash -c '
> > +    sleep 3 &&
> > +    python manage.py migrate --noinput &&
> > +    python manage.py runserver 0.0.0.0:8077'
> > +  volumes:
> > +    - .:/src
> > +  environment:
> > +    PORT: 8077
> > +    DATABASE_URL: postgres://postgres@db:5432/postgres
> > +  ports:
> > +    - "8077:8077"
> > +  links:
> > +    - db
> > +dredd:
> > +  image: ccxcon_web
> > +  command: /bin/bash -c 'sleep 4 && dredd'
>
> This is causing the containers to close once dredd has finished executing
> after running docker-compose up. Can you remove the command here and call
> it from travis.yml directory, or figure out some other way to keep the
> containers from stopping?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mitodl/ccxcon/pull/7/files#r44711448>.
>